src/diagnostics/DiagnosticsScreen.js

```diff
@@ -31,10 +31,7 @@ class DiagnosticsScreen extends PureComponent<Props> {
 
     return (
       <Screen title="Diagnostics">
-        <RawLabel
-          style={styles.versionLabel}
-          text={`App version: v${DeviceInfo.getReadableVersion()}`}
-        />
+        <RawLabel style={styles.versionLabel} text={`App version: v${DeviceInfo.getVersion()}`} />
```

Comment:
Also remove the 'App version:' string as it will not be translated anyways.

Reply:
Done